modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -118,6 +118,12 @@ CREATE DATABASE blockdb OPTIONS {storeFormat: 'block'}
 Starting from 5.23, `block` format is the preferred format for Enterprise Edition due to its superior performance and scalability.
 Therefore, migrating all databases to `block` format is recommended to ensure optimal performance.
 
+[NOTE]
+====
+Be aware that changing the store format changes the built-in _id property and elementID of nodes and relationships.
```

Comment:
Should `_id` and `elementID` be in backticks?

Reply:
elementID - yes. 
_id property - I have no idea how it looks in the code. I couldn't find it.